### PR TITLE
Dobby

### PR DESCRIPTION
**Pull request by Vudthi**

### CODE REVIEW

**Review by zigen — CHANGES_REQUESTED** [review]
fix indents

**Review by zigen — APPROVED** [review]

File: input-counter/main.js [hunk 1]
```diff
@@ -1 +1,23 @@
...
+const plusButton = document.getElementById('plus')
+const minusButton = document.getElementById('minus')
+const resetButton = document.getElementById('reset')
+
+const calculate = operation => () => {
+  operation == '+'
+    ? (output += parseInt(numberInput.value))
+    : operation == '-'
+    ? (output -= parseInt(numberInput.value))
+    : (output = 0)
```
Comment (hrl7) [inline input-counter/main.js]: you seems too smart! but it's hard to read and understand.

**Review by hrl7 — CHANGES_REQUESTED** [review]
could you fix more readable?

File: input-counter/main.js [hunk 2]
```diff
@@ -1 +1,27 @@
...
+  } else {
+    output = 0
+    numberElement.textContent = output
+  }
+  numberInput.value = ''
+}
+
+plusButton.addEventListener('click', () => calculate('+'))
+minusButton.addEventListener('click', () => calculate('-'))
+resetButton.addEventListener('click', () => calculate(false))
```
Comment (hrl7) [inline input-counter/main.js]: when the `resetButton` clicked, do you need to `calculate` and do you have any reason to pass `false` ?

**Review by hrl7 — CHANGES_REQUESTED** [review]
when the resetButton clicked, do you need to calculate and do you have any reason to pass false ?

File: input-counter/main.js (hunk 2 above)
Comment (Vudthi) [reply input-counter/main.js]: We have changed the calculate to reset function.

**Review by hrl7 — APPROVED** [review]

**Pull request closed without merging**